yelp_package/Makefile

```diff
@@ -69,6 +70,23 @@ package_trusty: build_trusty_docker
 itest_trusty: package_trusty bintray.json
 	$(DOCKER_RUN_TRUSTY) /work/yelp_package/itest/ubuntu.sh paasta-tools_$(ACTUAL_PACKAGE_VERSION)_amd64.deb
 
+build_xenial_docker:
```

Comment:
At 3 times, it is a good excuse to dry this out if you can. I won't block a shipit though because make is scary amirite? :)